core/src/test/java/org/infinispan/test/ReplListener.java

```diff
@@ -259,7 +259,11 @@ private boolean isPrimaryOwner(VisitableCommand cmd) {
       @Override
       protected Object handleDefault(InvocationContext ctx, VisitableCommand cmd) throws Throwable {
          try {
-            // first pass up chain
+            if (!ctx.isOriginLocal() || (watchLocal && isPrimaryOwner(cmd))) {
+               infof("Delaying command %s", cmd);
```

Comment:
I am guessing this is not needed anymore? It was just to help reproduce the issue correct?

Reply:
Or if we do keep it, which it looks like we wanted, this should be debug

Reply:
We do need the delay, otherwise the test is kind of pointless -- although it would be much nicer to use `ControlledRpcManager` instead and unblock the requests immediately instead of sleeping.

I logged it at INFO leve because the other logs in `ReplListenerInterceptor` already use `infof()`, but I'll change it to `debugf()` everywhere.

Reply:
It was actually logging at debug level, only the method name stayed `infof()` :)